### ISSUE TITLE
Base network selection

### ISSUE THREAD

**Issue by DonaldKam** [shijieS/SST#47]
As there are more popular and high-efficient base networks, i.e. ResNetX, why select VGG as the feature extractor? As we know, it enjoys tremendous parameters that slow down the computation speed and requires increasing memory.

**Comment by shijieS** [comment]
Yes, the feature extractor can be ResNet, DenseNet, etc. Actually, this code is based on SSD whose feature extractor is VGG. Also, you can try other feature extractors. I think you can get better results.  Besides, we only employ the center of the box which is also the weakness of this network. There are lots of room form improvement.

**Comment by shijieS** [comment]
Yes, the feature extractor can be ResNet, DenseNet, etc. Actually, this code is based on SSD whose feature extractor is VGG. Also, you can try other feature extractors. I think you can get better results. Besides, we only employ the center of the box which is also the weakness of this network. There are lots of room for improvement.

**Comment by DonaldKam** [comment]
Thank you and you remind me of another question. What is the center of the box? Coordinate? If not, will that be a patch cropped from a bbox?

**Comment by shijieS** [comment]
Features at the center of the box in the corresponding feature map.
